gradle/scripts/spotless.gradle

```diff
@@ -33,7 +33,7 @@ spotless {
 
   //sets a license header, removes unused imports and formats conforming to the scala fmt formatter
   scala {
-    scalafmt()
+    scalafmt('3.0.0').configFile('.scalafmt.conf')
```

Comment:
Could be updated to 3.4.3. Otherwise: Would no specific version work as well? Because, then we wouldn't need to take care of eventually upcoming updates.

Reply:
Removed the tag, we're now up to `3.0.8`